ui/pages/swaps/searchable-item-list/index.scss

```
@@ -1,39 +1,39 @@
 .searchable-item-list {
-  background: var(--white);
+  background: var(--color-background-default);
   width: 100%;
   position: relative;
 
   &__search {
     > div {
       border: none;
-      border-bottom: 1px solid var(--Grey-100);
+      border-bottom: 1px solid var(--color-border-default);
       border-radius: 0;
       height: 55px;
       font-size: 12px;
 
       input {
         @include H6;
 
-        color: var(--Grey-500);
+        color: var(--color-text-muted);
```

Hmm is this the color of the input? If it's a user entered text we should be using the highest contrasting text which is text-default(Grey900) not sure why they were using Grey500 here.
```suggestion
        color: var(--color-text-default);
```